blurb_it/util.py

```diff
@@ -35,6 +36,23 @@
     return request_session.get("username") and request_session.get("token")
 
 
+async def get_csrf_token(request):
+    request_session = await get_session(request)
+    try:
+        return request_session["csrf"]
+    except KeyError:
+        request_session["csrf"] = csrf = create_csrf_token()
+        return csrf
```

Comment:
This could be written with the walrus operator, depending on the version of Python that blurb_it runs on, but according to runtime.txt, it seems to be 3.7.